flowetl/requirements.in

```diff
@@ -0,0 +1,18 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# FlowETL requirements (input to pip-compile)
+#
+# Note: if we used pip-tools to compile directly from setup.py,
+# we wouldn't be able to specify the constraints file required for installing airflow
+
+# Airflow and python versions must match the versions in the base docker image
+-c https://raw.githubusercontent.com/apache/airflow/constraints-${AIRFLOW_VERSION}/constraints-${PYTHON_VERSION}.txt
+
+# Specifying editable install of flowetl here so we get its dependencies, but we don't want to actually install it
+# so need to specify '--unsafe-package flowetl' option to pip-compile
+-e ./flowetl
```

Comment:
I don't think it makes any difference in this case because we skip flowetl when compiling (and we'd have to do that with a non-editable install as well - a non-editable install from a local path is still not hashable, and would therefore prevent requirements.txt from being `pip install`-able).